src/BizHawk.Emulation.Cores/Consoles/Nintendo/GBA/MGBAHawk.IMemoryDomains.cs

```diff
@@ -31,7 +31,7 @@ private void CreateMemoryDomains(int romsize)
 			mm.Add(_vram = new MemoryDomainIntPtr("VRAM", le, IntPtr.Zero, 96 * 1024, true, 4));
 			mm.Add(_oam = new MemoryDomainIntPtr("OAM", le, IntPtr.Zero, 1024, true, 4));
 			mm.Add(_rom = new MemoryDomainIntPtr("ROM", le, IntPtr.Zero, romsize, true, 4));
-			mm.Add(_sram = new MemoryDomainIntPtr("SRAM", le, IntPtr.Zero, 0, true, 4)); // size will be fixed in wireup
+			mm.Add(_sram = new MemoryDomainIntPtr("SaveRAM", le, IntPtr.Zero, 0, true, 4)); // size will be fixed in wireup

```

Comment:
Reason for this change?

Reply:
'SRAM' as short for savedata seems misleading to me, especially since GBA uses separate save types, one of which is SRAM (static RAM in that context, of course).

Reply:
SRAM means save ram and static ram both. That ship has sailed. To make matters worse, WRAM is usually DRAM which may or may not be SRAM depending on whether it's battery backed, and may be called SRAM in any event. The words have no clear meaning so I don't care which one is used, but if there's a reason not to change it then it's better than a reason for changing it, unless it's to make it match other cores more closely in case this is an odd man out

Reply:
Changing it will break pre-existing .wch files, and maybe some other files I can't think of

Reply:
I'm pretty sure that it was my (possibly poor) decision to use "SRAM" here when originally brining in our first GBA core, and then I carried through the names identically for back compat since then.

Reply:
I don't think backwards compatibility with `.wch` files is a huge concern, since recent changes have very likely broken backwards compatibility for a few versions; feos's change that exposed the implementation detail that mGBA's internal  SaveRAM domain is not always defined (and zeromus's fix that really just worked around this fact).